### DISCUSSION TITLE
Internationalisation (i18N) support (and redifintion of hardcoded text blocks)

### DISCUSSION THREAD

**Discussion by RaphaelBossek** [nhost/nhost#2619] [Ideas]
The whole nhost project is English only and with hard coded text blocks everywhere. Is there any plan to support i18n and ability to change the predifined text blocks?

**Comment by dbarrosop** [comment]
What are you referring to in particular?

**Comment by RaphaelBossek** [comment]
My fault. I've studied the https://github.com/nhost/nhost/tree/main/examples/react-apollo implementation and was lost (in translation). My impression was, that the UI is part of nhost core API but it's not.
